python/singa/command.py

```diff
@@ -21,12 +21,16 @@
 
 To use this script, user sudo install these dependencies: flask pillow and protobuf
 '''
+from __future__ import print_function
 
+from future import standard_library
+standard_library.install_aliases()
+from builtins import str
 import sys, glob, os, random, shutil, time
 from flask import Flask, request, redirect, url_for
 import numpy as np
-import ConfigParser
-import urllib, traceback
+import configparser
```

Comment:
in py2 configparser comes with python.
ok. so now we just add it as an dependent lib.